baseframe/forms/validators.py

```diff
@@ -299,7 +300,23 @@ class NotEqualTo(_Comparison):
     def compare(self, value, other):
         return value != other
 
+class IsEmoji(object):
+    """
+    Validate that field.data does not belong to a public email domain.
+    If the domain lookup fails and mxsniff raises ``MXLookupException``, this validator
+    will still pass, as we expect that most domains are not public email domains.
+
+    :param message:
+        Error message to raise in case of a validation error.
+    """
+    def __init__(self, message=None):
+        self.message = message or _(u'This is not a valid emoji.')
```

Comment:
Message should be in double quotes. Our convention is to use double quotes for UI strings that need translation, and single quotes for tokens where the exact string must be preserved.

Reply:
@qurbat this is not fixed.

Reply:
@iambibhas fixed now